modules/cluster_estimation/cluster_estimation.py

```diff
@@ -0,0 +1,304 @@
+"""
+Take in bounding box coordinates from Geolocation and use to estimate landing pad locations.
+Returns an array of classes, each containing the x coordinate, y coordinate, and spherical 
+covariance of each landing pad estimation.
+"""
+
+import sklearn
+import sklearn.datasets
+import sklearn.mixture
+import numpy as np
+
+
+from ..object_in_world import ObjectInWorld
+from ..detection_in_world import DetectionInWorld
+
+
+class ClusterEstimation:
+    """
+    Estimate landing pad locations based on bounding box coordinates
```

Comment:
Explain the basic theoretical background for developers using this class (e.g. the estimation is implemented by clustering groups of landing pads using ...).